akka-sample-cluster-java/src/main/resources/logback.xml

```diff
@@ -4,7 +4,7 @@
         logging config, see the Akka docs: https://doc.akka.io/docs/akka/2.6/typed/logging.html#logback -->
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>[%date{ISO8601}] [%level] [%logger] [%marker] [%thread] - %msg MDC: {%mdc}%n</pattern>
+            <pattern>[%date{ISO8601}] [%level] [%logger] [%marker] [%thread] - %msg%n</pattern>
```

Comment:
Perhaps we could still show how to include a single MDC value? Does `[%X{akkaSource}]` still work for that?

Reply:
For the cluster samples that could be nice to start with as it shows which system it is when you run several in the same JVM